packages/expo-image/src/Image.types.ts

```diff
@@ -89,7 +89,7 @@ export type ImageDecodeFormat = 'argb' | 'rgb';
  * Some props are from React Native Image that Expo Image supports (more or less) for easier migration,
  * but all of them are deprecated and might be removed in the future.
  */
-export interface ImageProps extends Omit<ViewProps, 'style'> {
+export interface ImageProps extends Omit<ViewProps, 'style' | 'children'> {
```

Comment:
As this changes types, you also need to run `yarn build` in `packages/expo-image` to rebuild TS definitions in the build folder

Reply:
Ok on it! Thank you.